src/core/mutation.ts

```diff
@@ -84,6 +85,7 @@ export class Mutation<
   state: MutationState<TData, TError, TVariables, TContext>
   options: MutationOptions<TData, TError, TVariables, TContext>
   mutationId: number
+  meta?: MutationMeta
```

Comment:
```suggestion
  meta: MutationMeta | undefined
```

we want to always pass it to the constructor, but it might be `undefined`. With the current notation, we could theoretically not pass it to the constructor and TypeScript would be fine with it